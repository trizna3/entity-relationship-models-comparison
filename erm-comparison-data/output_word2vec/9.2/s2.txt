tile-tiles
weather-weathers
units-units
boards-boards
fuel type-boards;weathers
tile type-unit
games-games
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-unit-tiles
REBIND_MN_TO_1NN1:S-boards-weathers
