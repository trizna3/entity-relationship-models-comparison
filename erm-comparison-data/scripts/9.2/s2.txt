entity set games {
	"id"
};

entity set boards {
	"row"
	"column"
};

entity set tiles {
	"type"
	"unit"
};

entity set units {
	"name"
	"health"
	"attack strength"
	"defense strength"
	"point count"
};

entity set weathers {
	"weather 1"
	"weather 2"
	"weather 3"
};

entity set "move count" {
};

entity set fuel {
	"type"
};

association between 
{
	games ONE
	boards ONE
}{};

association between 
{
	tiles MANY
	boards ONE
}{};

association between 
{
	tiles ONE
	units ONE
}{};

association between 
{
	boards MANY
	weathers MANY
}{};

association between 
{
	units MANY
	fuel ONE
}{};

generalization from "move count" to weathers;