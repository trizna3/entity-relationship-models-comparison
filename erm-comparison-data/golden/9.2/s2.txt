games-games
boards-boards
tile-tiles
tile type-
weather-weathers
fuel type-fuel
units-units